### ISSUE TITLE
Cannot open dialog immediately after swal.close()

### ISSUE THREAD

**Issue by fracz** [t4t5/sweetalert#492]
Consider this code:

``` js
swal.close();
swal('The dialog');
```

It does not show "The dialog".

**Comment by senseluo** [comment]
same here, currently I use setTimeout to show dialog box.
```
swal.close();
setTimeout(function () {
  swal('The dialog');
}, 500)
```

**Comment by fracz** [comment]
Consider using [sweetalert2](https://github.com/limonte/sweetalert2) - it does not suffer from this.

**Comment by senseluo** [comment]
thanks @fracz
sweetalert2 is awesome and looks it all fit my needs.
thank you guys

**Comment by t4t5** [comment]
Fixed in 2.0. Make sure you read the [upgrade guide](https://sweetalert.js.org/guides/#upgrading-from-1x)!
